### PR TITLE
First playbook.  Last use of passwords.

### PR DESCRIPTION
**Pull request by pallenSL**
First Ansible:
Config points to hosts.
Hosts is one group of "nodes" to be treated all the same.
One easy playbook to remove the need to enter passwords each time.
And try to format the README.md

### CODE REVIEW

File: dataCenter/ansible/README.md [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+Install ssh key
+------------------------
+Set us up for password-free use (i.e. generate a ssh key and install it on the nodes)
+```
+$ ssh-keygen -t rsa -b 2048
+$ ssh-keyscan auth comms sessions gateway frontend >> ~/.ssh/known_hosts
+
+$ cd /vagrant
+$ ansible-playbook nodes-ssh-addkey.yml --ask-pass
+password: isanopensecret
```
Comment (jebbinSCL) [inline dataCenter/ansible/README.md]: Why is this an open secret? Or is it that a secure password is meant to be used here as a replacement?

Comment (pallenSL) [inline dataCenter/ansible/README.md]: You can guess vagrant's password...

Comment (jebbinSCL) [inline dataCenter/ansible/README.md]: Ah ok.

**Comment by jebbinSCL** [comment]
:shipit:

**Pull request merged**